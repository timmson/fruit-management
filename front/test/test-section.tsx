import React from "react"
import renderer from "react-test-renderer"
import Section from "../src/section"
import Context from "../src/context"

describe("Section should", () => {

	test("return section", () => {
		const entity = {name: "name", description: "descr"}

		const context = []

		const component = renderer.create(
			<Context.Provider value={context}>
				<Section value={entity}/>
			</Context.Provider>
		)

		expect(component.toJSON()).toMatchSnapshot()

		component.unmount()
	})

})