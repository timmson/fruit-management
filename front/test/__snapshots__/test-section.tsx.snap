// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section should return section 1`] = `
<td
  className="pt-2 text-center align-bottom"
  onClick={[Function]}
  style={
    {
      "width": "15%",
    }
  }
>
  <p
    className="av"
  >
    <img
      alt="name"
      src="img/admin_name.gif"
    />
    <br />
    descr
  </p>
</td>
`;
